packages/frontend/src/components/wallet/TokenBox.js

```diff
@@ -104,15 +101,12 @@ const StyledContainer = styled.div`
     }
 
     .balance {
-        margin-left: ${({IS_USN}) =>(IS_USN ? 0 : 'auto')};
+        margin-left: auto !important;
```

Comment:
nit: can we please opt to use `&` for added specificity when needed vs `!important`?

if `.balance` is already getting styled up above maybe we can/should rename it here to avoid needing more specificity all together?

Reply:
after getting rid of the swap usn button inline under your portfolio, realized this isn't even needed. thanks for the catch!